headerCheck/factory/nodesCoordinator.go

```diff
@@ -45,23 +46,22 @@ func CreateNodesCoordinator(
 
 	initialEpoch := uint32(0)
 	arguments := nodesCoordinator.ArgNodesCoordinator{
-		Epoch:                      initialEpoch,
-		ShardConsensusGroupSize:    int(networkConfig.ShardConsensusGroupSize),
-		MetaConsensusGroupSize:     int(networkConfig.MetaConsensusGroup),
-		Marshalizer:                coreComp.Marshaller,
-		EpochStartNotifier:         &disabled.EpochStartNotifier{},
-		BootStorer:                 &disabled.Storer{},
-		Hasher:                     coreComp.Hasher,
-		NbShards:                   networkConfig.NumShardsWithoutMeta,
-		EligibleNodes:              eligibleValidators,
-		WaitingNodes:               waitingValidators,
-		SelfPublicKey:              publicKeyBytes,
-		ConsensusGroupCache:        &disabled.NodesCoordinatorCache{},
-		WaitingListFixEnabledEpoch: enableEpochsConfig.WaitingListFixEnableEpoch,
-		ChanStopNode:               make(chan endProcess.ArgEndProcess),
-		NodeTypeProvider:           &disabled.NodeTypeProvider{},
-		Shuffler:                   nodeShuffler,
-		ShuffledOutHandler:         &disabled.ShuffledOutHandler{},
+		Epoch:                   initialEpoch,
+		ShardConsensusGroupSize: int(networkConfig.ShardConsensusGroupSize),
+		MetaConsensusGroupSize:  int(networkConfig.MetaConsensusGroup),
+		Marshalizer:             coreComp.Marshaller,
+		EpochStartNotifier:      &disabled.EpochStartNotifier{},
+		BootStorer:              &disabled.Storer{},
+		Hasher:                  coreComp.Hasher,
+		NbShards:                networkConfig.NumShardsWithoutMeta,
+		EligibleNodes:           eligibleValidators,
+		WaitingNodes:            waitingValidators,
+		SelfPublicKey:           publicKeyBytes,
+		ConsensusGroupCache:     &disabled.NodesCoordinatorCache{},
+		ChanStopNode:            make(chan endProcess.ArgEndProcess),
+		NodeTypeProvider:        &disabled.NodeTypeProvider{},
+		Shuffler:                nodeShuffler,
+		ShuffledOutHandler:      &disabled.ShuffledOutHandler{},
 	}
```

Comment:
we need to set here alse the enable epochs handler
i suggest to get rig of `createArgsNodesShuffler` and create enableEpochsHandler separately, with the necessary enable epochs config; also set custom config for multisiger:

```
enableEpochsConfig.EnableEpochs.BLSMultiSignerEnableEpoch = []config.MultiSignerConfig{
           {
               EnableEpoch: 0,
               Type:        "no-KOSK",
           },
           {
               EnableEpoch: 3,
               Type:        "KOSK",
           },
 }
```